accounts-service/build.gradle

```diff
@@ -0,0 +1,44 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '5.2.0'
+    id 'application'
+}
+
+application {
+    mainClassName = 'AccountsService'
+}
+
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    /**
+     * This enables using custom types with jersey requests
+     * https://mvnrepository.com/artifact/org.glassfish.jersey.media/jersey-media-json-jackson
+     * */
+    compile 'org.glassfish.jersey.media:jersey-media-json-jackson:2.17'
```

Comment:
It might make more sense that `infrastructure` module will have jackson in its dependencies. Then whoever depends on infra will get jackson

Reply:
leaving this as is

Reply:
didn't change this